snippets/coreos-layering-configuring-on-pause.adoc

```diff
@@ -0,0 +1,22 @@
+// Text snippet included in the following modules:
+//
+// * modules/coreos-layering-configuring-on.adoc
+// * modules/coreos-layering-configuring-on-modifying.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+Making certain changes to a `MachineOSConfig` object triggers an automatic rebuild of the associated custom layered image. You can mitigate the effects of the rebuild by pausing the machine config pool where the custom layered image is applied as described in "Pausing the machine config pools." For example, if you want to remove and replace a `MachineOSCOnfig` object, pausing the machine config pools before making the change prevents the MCO from reverting the associated nodes to the base image, reducing the number of reboots needed. 
```

Comment:
thought: By pausing the pool, the newly-built image will not be rolled out to the nodes after completion. However, the build will still run regardless of whether the pool is paused or not.